netbox/extras/api/views.py

```diff
@@ -273,10 +273,9 @@ def post(self, request, pk):
             raise RQWorkerNotRunningException()
 
         if input_serializer.is_valid():
-            Job.enqueue(
-                run_script,
+            ScriptJob = import_string("extras.jobs.ScriptJob")
```

Comment:
I think this is needed to avoid a circular import, however it should be fine to import `ScriptJob` directly (`from extras.jobs import ScriptJob`) at this point.

Reply:
This is a relict from using `BackgroundJob` for syncing data sources to avoid circular imports. For `ScriptJob` this is not necessary, so I've changed as requested in 7f0a4e3c34c06891d225535c36e3389fa6a8705d.